### PR TITLE
(obsolete)[UI]: Refactors meshery docs footer to match with Layer5 and Meshery websites.

### LINKED ISSUE #4767: [Docs]  Footer modification
#### Current State

Changes required on https://docs.meshery.io/ :
- we don't have layer5 discuss icon in docs footer
- The slack icon needs to be changed.
- the calendar icon doesn't have a hover effect.

#### Desired State

- add the layer5 discuss forum icon in docs footer.
- The slack icon needs to be modified to what we have on the [layer5](https://layer5.io/) website footer. You could checkout [#613](https://github.com/meshery/meshery.io/pull/613) to see how to add a hover effect to the slack icon.
- add a hover effect for the calendar icon.

**If there are still some doubts feel free to comment here or discuss it on slack.**

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Nik-09**
**Description**

This PR fixes #4767 

**Notes for Reviewers**

Modifies footer section of docs.meshery.io

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

Proof that the changes are correct:

https://user-images.githubusercontent.com/57531197/147380019-ff324b59-a7f3-493a-8049-75b44f8fea7f.MP4

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ba48ad72f4c3b3415d25fff1bc67a0288f9f4694 at: https://61c4bf96e6b481e1340aad52--meshery-docs-preview.netlify.app

**Comment by sayantan1413** [comment]
Hey, @Nik-09 whenever the PR is ready for review remove the label pr/draft.

**Comment by l5io** [comment]
🚀 Preview for commit 7b876d367808c433fd5f424da5c84dd795287d97 at: https://61c6c8fdb5f4a268a56c7dcb--meshery-docs-preview.netlify.app

**Review by Nik-09 — COMMENTED** [review]
@sayantan1413 Currently I do not have rights to add/remove label -- can you please remove it, as currently the PR is ready for review.
Thank you.

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Comment by Nik-09** [comment]
@Abhijay007 @sayantan1413 -- Do I have to add anything else in order to merge this PR ?
Thank you.

**Comment by Abhijay007** [comment]
> @Abhijay007 @sayantan1413 -- Do I have to add anything else in order to merge this PR? Thank you.

It's LGTM I don't think that you need to add anything else !!

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Good stuff @Nik-09, just some minor changes. Could you please change the default color of the icons to white, increase the size just a little bit and add some space between the icons.

**Comment by Nik-09** [comment]
Hi @warunicorn19 @leecalcote , I need a suggestion from you.
As I am working on https://github.com/meshery/meshery/issues/4767, I need to know that which one of the following will be better:
Same background with white icons (as earlier).
Black background with grey icons (as in Layer5 and Meshery.io website).
Looking forward to your feedback :)

**Comment by leecalcote** [comment]
@Nik-09 will you take a quick screenshot of the docs.meshery.io footer using the same off-black color used in the meshery.io footer? This will help in evaluating the potential change. The off-black color should be the Dark Jungle Green listed on layer5.io/brand.

**Comment by Nik-09** [comment]
@leecalcote 
please take a look.

image

**Comment by l5io** [comment]
🚀 Preview for commit 93a719832682b2f48c37eeaec14bae32903494a5 at: https://61dea07e78f171f08dc5647d--meshery-docs-preview.netlify.app

**Review by Nik-09 — COMMENTED** [review]
@warunicorn19 @leecalcote Please take a look.

**Comment by leecalcote** [comment]
Please add a more descriptive PR title, including `[UI]` at the beginning of the title. This saves maintainers time when reviewing PRs. Thanks!

**Comment by Nik-09** [comment]
> Please add a more descriptive PR title, including [UI] at the beginning of the title. This saves maintainers time when reviewing PRs. Thanks!

@leecalcote Done

File: docs/assets/css/main.css [hunk 1]
```diff
@@ -18671,34 +18671,33 @@ footer {
...
-.social__link--youtube:hover {
-    color: #ff0000;
+.social__link--calendar:hover {
+    fill: #0073cf;
 }
 
-.social__link--docker:hover {
-    color: #2496ed;
+.footer {
+    background-color: #222222;
```
Comment (warunicorn19) [inline docs/assets/css/main.css:18700]:
```suggestion
    background-color: #1E2117;
```

**Comment by l5io** [comment]
🚀 Preview for commit 85fa548628c9439d9de366dad5d249a31f7d5a1b at: https://61e445fbd5eabef1b499693b--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4c38ed13b963aace86c8168e28f60a9a89472d73 at: https://61e45e9fa71e1e288573aee8--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
Calendar icon is missing the + symbol
Screen Shot 2022-01-16 at 12 14 38 PM
.

**Comment by Nik-09** [comment]
> Calendar icon is missing the + symbol Screen Shot 2022-01-16 at 12 14 38 PM .

True, I removed that logo because of the following reasons:
1. Now the logo is matching with [meshery.io](https://meshery.io/) calendar logo in footer section hence it provides consitency.
2. It is new and clear in comparison to previous logo.

Looking forward to your point.
Thank you

**Comment by leecalcote** [comment]
Thanks for explaining this @Nik-09. I suspect that whoever made the meshery.io footer accidentally forgot to change the color of the internal part of that icon's svg (I haven't looked, but just a guess). I think the calendar icon is too generic when it is more or less only a rounded rectangle. It'd be nice to either include the "+" or find another calendar icon. You're right that we want to be consistent across all uses of this icon. 👍

**Comment by leecalcote** [comment]
@Nik-09, hopefully, you're able to colorize the "+" into the icon, and then help drive consistency across the rest of the site footers. Let me know if I can aid in updating the icon or finding a new one.

**Comment by Nik-09** [comment]
Thank you @leecalcote, I've edited the current svg and added "+" in it.
Please take a look,
image

Initially gray as similar to other icons and on hover, it will be colored blue(#0073cf).
**Notes:**
Please provide feedback or confirm whether this is ready to be updated on other platforms(meshery.io).

Thank you.

**Comment by l5io** [comment]
🚀 Preview for commit b90f9f433db7a319e6ef90cfb26df570cf6b0a34 at: https://61f6271e290c89788d304da8--meshery-docs-preview.netlify.app

**Comment by Nik-09** [comment]
Closing this pull request because of some problems regarding signing off, the changes are reflected on new PR https://github.com/meshery/meshery/pull/5017

Thank you

**Comment by leecalcote** [comment]
As you open a new PR, note that the "+" either isn't symmetric or isn't centered.

**Pull request closed without merging**